gtsfm/scene_optimizer.py

```diff
@@ -202,43 +200,7 @@ def create_computation_graph(
         keypoints_graph_list = dask.delayed(lambda x, y: (x, y))(keypoints_graph_list, auxiliary_graph_list)[0]
         auxiliary_graph_list = []
 
-        # ensure cycle consistency in triplets
-        # TODO: add a get_computational_graph() method to ViewGraphOptimizer
-        # TODO(johnwlambert): use a different name for variable, since this is something different
-        i2Ri1_graph_dict, i2Ui1_graph_dict, v_corr_idxs_graph_dict, rcc_metrics_graph = dask.delayed(
-            cycle_consistency.filter_to_cycle_consistent_edges, nout=4
-        )(
-            i2Ri1_graph_dict,
-            i2Ui1_graph_dict,
-            v_corr_idxs_graph_dict,
-            two_view_reports_dict,
-            EdgeErrorAggregationCriterion.MEDIAN_EDGE_ERROR,
-        )
-        metrics_graph_list.append(rcc_metrics_graph)
-
-        def _filter_dict_keys(dict: Dict[Any, Any], ref_dict: Dict[Any, Any]) -> Dict[Any, Any]:
-            """Return a subset of a dictionary based on keys present in the reference dictionary."""
-            valid_keys = list(ref_dict.keys())
-            return {k: v for k, v in dict.items() if k in valid_keys}
-
-        if gt_cameras_graph is not None:
-            two_view_reports_dict_cycle_consistent = dask.delayed(_filter_dict_keys)(
-                dict=two_view_reports_dict, ref_dict=i2Ri1_graph_dict
-            )
-            metrics_graph_list.append(
-                dask.delayed(two_view_estimator.aggregate_frontend_metrics)(
-                    two_view_reports_dict_cycle_consistent,
-                    self._pose_angular_error_thresh,
-                    metric_group_name="cycle_consistent_frontend_summary",
-                )
-            )
-            auxiliary_graph_list.append(
-                dask.delayed(save_full_frontend_metrics)(
-                    two_view_reports_dict_cycle_consistent,
-                    image_graph,
-                    filename="cycle_consistent_frontend_full.json",
-                )
-            )
+        # TODO: compute 2-view report post view graph optimization
```

Comment:
can we do it in the same PR? or will it require too much refactoring?

Reply:
Ah, somehow missed this TODO, will do in same PR